ext/nnstreamer/tensor_filter/tensor_filter_executorch.cc

```diff
@@ -141,7 +120,9 @@ executorch_subplugin::getEmptyInstance ()
 void
 executorch_subplugin::cleanup ()
 {
-  g_free (model_path);
+  if (model_path)
```

Comment:
unnecessary cond? (g_free() internally checks null case.)

Reply:
Updated it, PTAL